Gemfile

```diff
@@ -254,7 +254,7 @@ end
 gem 'webpacker', '5.4.4'
 
 gem 'developer_portal', path: 'lib/developer_portal'
-gem 'unicorn', require: false, group: %i[production preview]
```

Comment:
Question: what about `staging`? Will there be no more distinction between that and production anymore?

Reply:
There is no distinction indeed. Staging uses `Rails.env == 'production'`

The only difference is `BUGSNAG_RELEASE_STAGE`, which is set to `staging` in staging, to distinguish the bug reports in BugSnag.